pkg/yamlprocessor/yamlprocessor_test.go

```diff
@@ -96,7 +103,17 @@ foo:
 			wantErr: false,
 		},
 		{
-			name: "given a array path with wildcard",
+			name: "given an entire array path",
```

Comment:
nits, It may not relate to the change of this PR but I think this test name should be `given a path to an entire array` (also all other test cases' name).

Reply:
My bad, I just missed the mean of this test name 🙏  No need to make any change here 👍